base/bucket_n1ql.go

```diff
@@ -21,11 +22,12 @@ import (
 	"gopkg.in/couchbase/gocb.v1"
 )
 
-const KeyspaceQueryToken = "$_keyspace" // Token used for bucket name replacement in query statements
-const MaxQueryRetries = 30              // Maximum query retries on indexer error
-const IndexStateOnline = "online"       // bucket state value, as returned by SELECT FROM system:indexes.  Index has been created and built.
-const IndexStateDeferred = "deferred"   // bucket state value, as returned by SELECT FROM system:indexes.  Index has been created but not built.
-const IndexStatePending = "pending"     // bucket state value, as returned by SELECT FROM system:indexes.  Index has been created, build is in progress
+const KeyspaceQueryToken = "$_keyspace"    // Token used for keyspace name replacement in query statement. The replacement will be an escaped keyspace.
+const KeyspaceAlias = "queryKeyspaceAlias" // Keyspace alias set for the keyspace in FROM statements in queries
```

Comment:
Given this name will appear in bucket index definitions as well as logged query statements, I wonder if it's a bit too generic.  Even sgKeyspaceAlias might be better, while still being descriptive.